packs/pathfinder-bestiary-2/scythe-tree.json

```diff
@@ -195,7 +195,33 @@
                     "remaster": false,
                     "title": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "domain": "all",
+                        "key": "RollOption",
+                        "option": "woodland-ambush",
+                        "toggleable": true
+                    },
+                    {
+                        "key": "BaseSpeed",
```

Comment:
Used `BaseSpeed` as it doesn't grant a bonus as it does for Tree That Weeps

Reply:
Actually, maybe I should remove the BaseSpeed. Don’t think that part is suitable for automation.